lib/rubocop/cop/rspec/multiple_expectations.rb

```diff
@@ -89,12 +84,12 @@ def example_with_aggregate_failures?(example_node)
           node_with_aggregate_failures = find_aggregate_failures(example_node)
           return false unless node_with_aggregate_failures
 
-          aggregate_failures?(node_with_aggregate_failures)
+          aggregate_failures?(node_with_aggregate_failures, :true_type?.to_proc)
         end
 
         def find_aggregate_failures(example_node)
           example_node.send_node.each_ancestor(:block)
-            .find { |block_node| aggregate_failures_present?(block_node) }
+            .find { |block_node| aggregate_failures?(block_node, ANYTHING) }
```

Comment:
A new addition to `rubocop-ast` made it possible to pass in `true` and `_`-alike's.
That was it became possible to collapse two node matchers into one, since they only had that distinction.

One is used to match:
```ruby
describe aggregate_examples: true do
# or
describe :aggregate_examples do
```

the other any of:
```ruby
describe aggregate_examples: false do
# or
describe aggregate_examples: true do
# or
describe :aggregate_examples do
```